### PR TITLE
fix: enable lazy load for footer image

### PR DESCRIPTION
**Pull request by mmataja**
[TICKET](https://smg-au.atlassian.net/browse/DM-2885)

## Motivation and context
Enable lazy loading for app images in footer.

## Before
App images would load on page load.

## After
Footer app images are loaded when in viewport.

### CODE REVIEW

File: src/components/navigation/footer/Apps.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ const FooterApps: FC<Props> = ({ config }) => {
                 alt={'Googleplay Icon'}
                 width="136px"
                 height="40px"
+                loading="lazy"
```
Comment (lkappeler) [inline src/components/navigation/footer/Apps.tsx:38]: Does this work if the resource is inlined?

Comment (mmataja) [reply src/components/navigation/footer/Apps.tsx:38]: Not sure. Let me test it and I let you know. But is that relevant for this case?

Comment (lkappeler) [reply src/components/navigation/footer/Apps.tsx:38]: If it works the same way as loading an image from a source no what might be is that it has no effect as those images are inlined with `data:image...` and I assume the intention of the task is to improve the page performance?

The docs mention something regarding network and storage load
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#loading

Comment (mmataja) [reply src/components/navigation/footer/Apps.tsx:38]: Thanks!
I tested it on listings and it does work with inlined resource as well.

**Review by lkappeler — APPROVED** [review]

**Pull request merged**